composer.json

```diff
@@ -112,6 +112,9 @@
             },
             "drupal/dynamic_entity_reference": {
                 "entityQuery reference JOINs should specify target_type (https://www.drupal.org/project/dynamic_entity_reference/issues/3120952#comment-14141038)": "https://www.drupal.org/files/issues/2021-06-22/entityquery-reference-joins-should-specify-target_type-3120952-24.patch"
+            },
+            "drupal/maestro": {
+                "Maestro task notification permission patch": "https://gist.githubusercontent.com/jekuaitk/5595756eb1aaf4da5e780761ac619877/raw/340e16d23a99c3f0bf1044d144de222883c2ae01/maestro_task_select_notification_role_permission.patch"
             }
```

Comment:
is there a reason patch is servers from a private github space? I see it was already added to the project, i think it would be better to serve patch it from there.

Reply:
Good point.

I suppose predicting the path could work - something like 
`https://raw.githubusercontent.com/OS2Forms/os2forms/3.14.2/patches/drupal/maestro/maestro_task_select_notification_role_permission.patch`

Reply:
can it be a relative path, something like '../patches/maestro_task_select_notification_role_permission.patch' ?

Reply:
If we use relative path, it would have to be relative from `DRUPAL_ROOT` in the installing Drupal project, i.e. `web/modules/contrib/os2forms/patches/drupal/maestro/maestro_task_select_notification_role_permission.patch`. This seems wrong to me. 

Can we use the private gist for now and then with the release an update to `https://raw.githubusercontent.com/OS2Forms/os2forms/3.14.2/patches/drupal/maestro/maestro_task_select_notification_role_permission.patch`?

Reply:
Yeah, i see your point. Since os2forms is just a module, it can be enabled on an installation, which does not necessarily have webroot starting from web

I got another idea, which in my opinion solves all the problems. How about adding this issue to drupal.org? In this way we make a contribution to project (and hopefully some feedback), clear reference to the task and permanent link to the patch-file which it not dependent on github structure?